parts/configure-swarm-cluster.sh

```diff
@@ -6,6 +6,7 @@ echo "starting swarm cluster configuration"
 date
 ps ax
 
+# **NOTE** Keep version numbers in sync with constants in cmd/info.go
 SWARM_VERSION="swarm:1.1.0"

```

Comment:
If we really want the "info/orchestrator" cmd to be in sync and up to date then this should be passed as inputs to the shell script files (and have a const in go code). Otherwise these values can easily go out of sync. Otherwise LGTM @dmitsh